src/dependency-manager/config/service-config.ts

```diff
@@ -61,4 +60,5 @@ export interface ServiceConfig extends ResourceConfig {
   replicas: number | string; // TODO:290:number
   scaling?: ScalingConfig;
   deploy?: DeploySpec;
+  termination_grace_period?: string;
```

Comment:
I had considered this, but felt like it was kind of wrong to silently override Kubernetes defaults. What if someone registers a cluster in the future where the default setting isn't actually 30s for some kind of good, technical reason with that cluster version? I think that we should just be falling back to the cluster's default, not setting our own blanket value. But I could also agree with setting a default because if people are using our cluster and a cluster upgrade changes that value when they don't have it set, we'd be changing the value on the user unexpectedly

Reply:
After thinking about this a little more, I think it would be safer for the user if we include a default and pin this value

Reply:
I thought it'd be better for our own documentation to be consistent. Especially if users don't even know that kubernetes is part of the equation.